bundler.d/aws.rb

```diff
@@ -0,0 +1,3 @@
+group :aws do
+  gem 'fog-aws'
```

Comment:
I'm not sure, it depends on fog-core though so I don't think it'll get any breaking changes if we're already pinning the others. Given it's a really new gem (24 days old) I'd just go without pinning